en/docs/deploy/overview-of-deployment-options.md

```diff
@@ -0,0 +1,138 @@
+
+##### NGINX configuration
+
+```nginx
+events {}
+
+http {
+   upstream backend {
+      server 127.0.0.1 max_fails=3 fail_timeout=30s;
+      server 127.0.0.2 max_fails=3 fail_timeout=30s;
+   }
+
+   server {
+      listen 80;
+      location / {
+            proxy_pass http://backend;
+            health_check uri=/health interval=5s;
+      }
+   }
+}
+```
+
+#### **3. Active-Passive**
+
+Primary server handles all traffic, backup only activates on failure. The backup server remains idle until the primary fails, ensuring you always have a failover target.
+
+When the primary server fails to send a response, the load balancer immediately redirects the request to backup server. This failover process is automatic and transparent to the client, occurring within milliseconds of detecting the failure. The backup server must be pre-configured with identical application code and dependencies.
+
+Nginx tracks failed requests against `max_fails` threshold and after reaching threshold, server is marked as unavailable for `fail_timeout` duration. And then keep sending request to one of the backup servers. Once a server is marked as unavailable, Nginx will not attempt to send requests to it until the `fail_timeout` period expires, ensuring that the backup server handles all incoming traffic consistently. If multiple backup servers are configured, Nginx will select the first available backup server in the order they are defined, maintaining the single-active-server principle of active-passive architecture.
+
+After `fail_timeout` period, Nginx attempts to route traffic back to primary server. If successful, primary server resumes active role and backup servers return to standby mode. This recovery process is gradual and intelligent - Nginx sends a small number of test requests to the recovered primary server before fully transitioning traffic back. If the primary server successfully handles these test requests without errors, it regains its active status and the backup server automatically transitions back to standby mode. However, if the primary server continues to fail during the recovery attempt, it remains marked as unavailable for another `fail_timeout` period, and the backup server continues to handle all traffic until the next recovery cycle.
+
+##### NGINX configuration
+
+```nginx
+events {}
+
+http {
+   upstream backend {
+      server 127.0.0.1 max_fails=3 fail_timeout=30s;
+      server 127.0.0.2 max_fails=3 fail_timeout=30s;
+   }
+
+   server {
+      listen 80;
+
+      location / {
+            proxy_pass http://backend;
+      }
+   }
+}
+```
+
+???+ Tip "Best Practice"
+    Ensure identical configurations across all instances and automate deployments for consistency.
+
+You can visit the following sections to get an understanding on the possible deployment and configuration options.
+
+* [Containerized Deployment](/deploy/containerized-deployment/introduction-to-containerized-deployment)
```

Comment:
Have the `Deply to Devant` as the first item and maintain the order in left nav here also.